docs/howto-mac.md

```diff
@@ -0,0 +1,61 @@
+# RLLIB SUMO Docker environment for MacOS
+
+## Step 1: Build your docker image
+
+Basic command to use:
+
+`bash docker-cmd-mac.sh --build`
+
+- The default tag name of this image is `tf-gpu-sumo-{today_date}`. You can also add option such as `--image-name={new_name}` to specify your own name;
+- The default path of the dockerfile to be used is `docker-image-mac`. You can adapt to your setting by using option `--image-folder`;
+- The default image building process does not use cache. You can enable it by adding the option `--cache`.
+
+## Step 2: Run a container of the built image
+
+`sudo bash docker-cmd-mac.sh --image-name=tf-gpu-sumo-2020-08-03 --run --no-gpu`
```

Comment:
Can you use `tf-gpu-sumo-{today_date}` instead of a specific date?

Reply:
Updated in commit 0d33ba4c2174b2d6729a47f2fe4ce99afd0f5c7d